src/appkit.cs

```diff
@@ -20196,9 +20326,10 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 	
 		[Export ("resizeFlags")]
 		nint ResizeFlags { get; }
-	
-		[Export ("keyDown:")]
-		void KeyDown (NSEvent  theEvent);
+
+		// Inherits from NSResponder
+		// [Export ("keyDown:")]
```

Comment:
Was this method just removed? If so, it's a breaking change

Reply:
Yes, but the interface inherits from NSResponder, which still has keyDown:, so my understanding was that this was ok. I think @dalexsoto and/or @mandel-macaque suggested I comment this one out like this.

According to the diffs, it was simply removed from NSWindow. And the header files and web docs don't document its existence in NSWindow at all.